code/modules/surgery/organs/stomach/stomach_ethereal.dm

```diff
@@ -50,27 +78,37 @@
 	switch(crystal_charge)
 		if(-INFINITY to ETHEREAL_CHARGE_NONE)
 			SEND_SIGNAL(carbon, COMSIG_ADD_MOOD_EVENT, "charge", /datum/mood_event/decharged)
+			carbon.clear_alert("ethereal_overcharge")
 			carbon.throw_alert("ethereal_charge", /atom/movable/screen/alert/emptycell/ethereal)
-			if(carbon.health > 10.5)
+			if(carbon.health > 10.5 && !(organ_flags & ORGAN_FAILING)) //special interaction for failiure
 				carbon.apply_damage(0.65, TOX, null, null, carbon)
+				damage += (0.5 * damagemodifer)
 		if(ETHEREAL_CHARGE_NONE to ETHEREAL_CHARGE_LOWPOWER)
 			SEND_SIGNAL(carbon, COMSIG_ADD_MOOD_EVENT, "charge", /datum/mood_event/decharged)
+			carbon.clear_alert("ethereal_overcharge")
 			carbon.throw_alert("ethereal_charge", /atom/movable/screen/alert/lowcell/ethereal, 3)
 			if(carbon.health > 10.5)
 				carbon.apply_damage(0.325 * delta_time, TOX, null, null, carbon)
 		if(ETHEREAL_CHARGE_LOWPOWER to ETHEREAL_CHARGE_NORMAL)
 			SEND_SIGNAL(carbon, COMSIG_ADD_MOOD_EVENT, "charge", /datum/mood_event/lowpower)
+			carbon.clear_alert("ethereal_overcharge")
 			carbon.throw_alert("ethereal_charge", /atom/movable/screen/alert/lowcell/ethereal, 2)
 		if(ETHEREAL_CHARGE_ALMOSTFULL to ETHEREAL_CHARGE_FULL)
+			carbon.clear_alert("ethereal_overcharge")
+			carbon.clear_alert("ethereal_charge")
 			SEND_SIGNAL(carbon, COMSIG_ADD_MOOD_EVENT, "charge", /datum/mood_event/charged)
 		if(ETHEREAL_CHARGE_FULL to ETHEREAL_CHARGE_OVERLOAD)
 			SEND_SIGNAL(carbon, COMSIG_ADD_MOOD_EVENT, "charge", /datum/mood_event/overcharged)
+			carbon.clear_alert("ethereal_charge")
 			carbon.throw_alert("ethereal_overcharge", /atom/movable/screen/alert/ethereal_overcharge, 1)
 			carbon.apply_damage(0.2, TOX, null, null, carbon)
+			damage += (1 * damagemodifer)
```

Comment:
Please either lower or remove the damage for this stage, as it is often very easy to ethereals to accidentally stray into the overload territory and I'd rather not heavily punish them for it.

Reply:
with the new changes to cyborg chargers and ethereals overcharge is less likely, also the variable damagemodifer is at 0.5, so this damage is 0.5 over time not 1.

Reply:
I still want what wesoda said to be done, because cyborg charger is not the only mean of charging.

Reply:
overload is punished by shooting bolts of lightning isn't it? if you want to do this you should give more reasoning for why it should be done

Reply:
Critical overcharge lasts 10 seconds tops before you vomit out lightning, i have tested this before and the damage from overcharge is insignificant, note that the damage is actually 0.25 because of the damage modifier

Reply:
> overload is punished by shooting bolts of lightning isn't it? if you want to do this you should give more reasoning for why it should be done

You're thinking of discharging, which is only possible in the CHARGE_DANGEROUS threshold. CHARGE_OVERLOAD - what we're talking about - is effectively the grace threshold between full charge and dangerous. It's relatively easy to fall into, be it from rechargers or not (although before this they were definitely the worst culprits), hence why we're trying to keep the damage for this threshold minimal.

The value they chose actually isn't too terrible, but in context with the other values for varying amounts of charge I don't think it makes sense. With the current values, an ethereal on an empty stomach will take half of much damage as one in the overload area, yet it's far easier to find yourself in overload than empty. I think the values should reflect the comparative difficulty of falling into a category. That being said, I'm definitely fine with the DANGEROUS area to deal a lot, cause it both makes sense and there are protections in place to prevent anyone from being in said category too long.

Reply:
the damage it does is very low and if you overcharge via apc you can just put the power back in. the times i have overcharged have been rare and far between, and it usually is dealt with within a minute, also considering that with more damage the faster charge is consumed meaning that you spend less time in overcharge, also the organ still heals during this time.